pkg/cli/democluster/demo_cluster_test.go

```diff
@@ -406,7 +406,7 @@ func TestTenantCapabilities(t *testing.T) {
 		if cap == tenantcapabilities.TenantSpanConfigBounds {
 			capValue = `{}`
 		}
-		expectedRows = append(expectedRows, []string{`2`, demoTenantName, `ready`, `shared`, cap.String(), capValue})
+		expectedRows = append(expectedRows, []string{`3`, demoTenantName, `ready`, `shared`, cap.String(), capValue})
```

Comment:
:D It is a rare day when we change something like this and a single test fails.

Reply:
I got lucky then.. :)